test/e2e/access_log_test.go

```diff
@@ -123,19 +138,40 @@ var _ = Describe("Access Log", func() {
 					gw,
 				}
 			})
-
 			It("can create string access logs", func() {
 				requestBuilder := testContext.GetHttpRequestBuilder().
 					WithPath("1").
+					WithQuery("foo=bar").
 					WithPostMethod()
 				Eventually(func(g Gomega) {
 					g.Expect(testutils.DefaultHttpClient.Do(requestBuilder.Build())).Should(matchers.HaveOkResponse())
 
 					logs, err := testContext.EnvoyInstance().Logs()
 					g.Expect(err).NotTo(HaveOccurred())
-					g.Expect(logs).To(ContainSubstring(`"POST /1 HTTP/1.1" 200`))
+					g.Expect(logs).To(ContainSubstring(`"POST /1?foo=bar HTTP/1.1" 200`))
 				}, time.Second*30, time.Second/2).Should(Succeed())
 			})
+			Context("Formatter extensions", func() {
+				BeforeEach(func() {
+					gw.GetOptions().GetAccessLoggingService().GetAccessLog()[0].GetFileSink().OutputFormat = &als.FileSink_StringFormat{
+						StringFormat: "%REQ(:PATH)%; %REQ_WITHOUT_QUERY(:PATH)%; %METADATA(ROUTE:foo-namespace)%\n",
+					}
+				})
+				It("can create formatted string access logs", func() {
+					requestBuilder := testContext.GetHttpRequestBuilder().
+						WithPath("1").
+						WithQuery("sensitive=data&needs=removed").
+						WithPostMethod()
+					Eventually(func(g Gomega) {
+						g.Expect(testutils.DefaultHttpClient.Do(requestBuilder.Build())).Should(matchers.HaveOkResponse())
+
+						logs, err := testContext.EnvoyInstance().Logs()
+						g.Expect(err).NotTo(HaveOccurred())
+						g.Expect(logs).To(ContainSubstring(`/1?sensitive=data&needs=removed; /1; {"bar-metadata":"greetings"}`))
```

Comment:
nit: second part of [above suggestion](https://github.com/solo-io/gloo/pull/9247/files#r1532149643)
```suggestion
						g.Expect(logs).To(ContainSubstring(`req: /1?sensitive=data&needs=removed`))
						g.Expect(logs).To(ContainSubstring(`req_without_query: /1`))
						g.Expect(logs).To(ContainSubstring(`metadata: {"bar-metadata":"greetings"}`))
```